paper/paper.bib

```diff
@@ -8,20 +8,17 @@
 
 @misc{manif,
 	Author = {Jeremie Deray and Joan Sol\`a},
-	Date-Added = {2019-01-09 19:46:45 +0100},
-	Date-Modified = {2019-01-09 22:35:53 +0100},
-	Howpublished = {\url{https://github.com/artivis/manif}},
+	url = {https://github.com/artivis/manif},
 	Month = {January},
 	Read = {0},
 	Title = {\texttt{manif}: a small {C++} header-only library for {L}ie theory.},
 	Year = {2019},
```

Comment:
```suggestion
	Year = {2019},}
```
`manif` is missing its closing parenthesis.

Reply:
fixed